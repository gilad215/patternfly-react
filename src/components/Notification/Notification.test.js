import React from 'react';
import { mount } from 'enzyme';
import { Notification } from './index';

test('Notification is working properly', () => {
  const component = mount(
    <Notification>
      <div>Child</div>
    </Notification>
  );
  expect(component.render()).toMatchSnapshot();
});

test('Notification Content is working properly', () => {
  const component = mount(
    <Notification.Content>
      <div>Child</div>
    </Notification.Content>
  );
  expect(component.render()).toMatchSnapshot();
});

test('Notification Info is working properly', () => {
  const component = mount(<Notification.Info />);

  expect(component.render()).toMatchSnapshot();
});

test('Notification Message is working properly', () => {
  const component = mount(
    <Notification.Message>
      <div>Child</div>
    </Notification.Message>
  );
  expect(component.render()).toMatchSnapshot();
});
