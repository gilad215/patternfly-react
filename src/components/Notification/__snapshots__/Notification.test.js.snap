// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification Content is working properly 1`] = `
<div
  className="drawer-pf-notification-content"
>
  <div>
    Child
  </div>
</div>
`;

exports[`Notification Info is working properly 1`] = `
<div
  className="drawer-pf-notification-info"
>
  <span
    className="date"
  >
    
  </span>
  <span
    className="time"
  >
    
  </span>
</div>
`;

exports[`Notification Message is working properly 1`] = `
<span
  className="drawer-pf-notification-message"
>
  <div>
    Child
  </div>
</span>
`;

exports[`Notification is working properly 1`] = `
<div
  className="drawer-pf-notification unread"
>
  <div>
    Child
  </div>
</div>
`;
